tests/test_frame.py

```diff
@@ -60,20 +92,6 @@ def test_headers_merging_none_returns_empty_dict(self, _tfs_file_x_pathlib, _tfs
         result = merge_headers(headers_left, headers_right, how=how)
         assert result == OrderedDict()  # giving None returns empty headers
 
-    def test_providing_new_headers_overrides_merging(self, _tfs_file_x_pathlib, _tfs_file_y_pathlib):
-        dframe_x = tfs.read(_tfs_file_x_pathlib)
-        dframe_y = tfs.read(_tfs_file_y_pathlib)
-
-        assert dframe_x.append(other=dframe_y, new_headers={}).headers == OrderedDict()
-        assert dframe_y.append(other=dframe_x, new_headers={}).headers == OrderedDict()
-
-        # we provide lsuffix (or rsuffix) since dframes have the same columns
-        assert dframe_x.join(other=dframe_y, lsuffix="_l", new_headers={}).headers == OrderedDict()
-        assert dframe_y.join(other=dframe_x, lsuffix="_l", new_headers={}).headers == OrderedDict()
-
-        assert dframe_x.merge(right=dframe_y, new_headers={}).headers == OrderedDict()
-        assert dframe_y.merge(right=dframe_x, new_headers={}).headers == OrderedDict()
-
```

Comment:
this test can stay, you only need to remove the lines until 67-73

Reply:
Indeed with `merge` back I will bring it back too

Reply:
and maybe add concat?

Reply:
Done!